katas/kata_adalid/lang.js

```diff
@@ -0,0 +1,112 @@
+const antMoves = (large = undefined, steps = undefined, sense = '') =>{
```

Comment:
HI Adalid, please rename folder to just `adalid` so it follow convention described in README file. Also, this folder should be under its corresponding kata folder